static/dbstructure.config.php

```diff
@@ -54,11 +54,40 @@
 use Friendica\Database\DBA;
 
 if (!defined('DB_UPDATE_VERSION')) {
-	define('DB_UPDATE_VERSION', 1349);
+	define('DB_UPDATE_VERSION', 1350);
 }
 
 return [
 	// Side tables
+	"gserver" => [
```

Comment:
Can you please move back this definition in the table name alphabetical order it was in before?

Reply:
See above.

Possibly we can change this foreign key creation in a way that we always perform this after the table had been created. But vurrently this is the only way. I don't like it as well.

Reply:
Related to [this comment](https://github.com/friendica/friendica/pull/8678#discussion_r429268068), I'd rather have a programmatic way of ordering table creation than relying on the implicit order of this array.

Reply:
Especially we could have some circular dependancies, I guess. Then it could get really funny :-)

Reply:
`mysqldump` has an option to create all the indices after tables have been created to avoid such an issue.